### DISCUSSION TITLE
bug: duplicate chat created when using Open WebUI as a search engine

### DISCUSSION THREAD

**Discussion by adan89lion** [open-webui/open-webui#7880] [Issues]
# Bug Report

## Installation Method

Docker

## Environment

- **Open WebUI Version:** v0.4.8
- **Ollama (if applicable):** 0.5.1

- **Operating System:** Linux Mint 22 Cinnamon (but reproducable on all desktop OSes)
- **Browser (if applicable):** Universal (reproducable in Chromium and Gecko based browsers)

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.
- [x] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

When deleting the current conversation created via search engine, it should return to a clean new chat.

## Actual Behavior:

Open WebUI restarted a conversation with the prompt provided in the search engine query.

## Description

**Bug Summary:**
I followed the  "Browser Search Engine" integration tip from the official [documentation](https://docs.openwebui.com/tutorials/integrations/browser-search-engine). When searching with `webui Why is the sky blue` in browser search bar, it correctly redirected to Open WebUI and started the conversation with the query. But when _deleting_ this conversation, Open WebUI will start a new `Why is the sky blue` conversation.

## Reproduction Details

**Steps to Reproduce:**
1. Correctly setup the search engine from the official [documentation](https://docs.openwebui.com/tutorials/integrations/browser-search-engine).
2. In the search bar, type `webui Why is the sky blue`
3. The browser redirects to my Open WebUI instance, asking the default model with the question `Why is the sky blue`
4. After the default model answered the query, delete the current conversation.
5. Open WebUI started another conversation with `Why is the sky blue`, instead of a clean new conversation.

## Logs and Screenshots

**Browser Console Logs:**
N/A

**Docker Container Logs:**
N/A

**Screenshots/Screen Recordings (if applicable):**
N/A

## Additional Information
N/A

**Comment by tjbck** [comment]
Not entirely sure what you're reporting here, visual aids would be tremendously helpful. Keep us posted!

**Reply by adan89lion** [reply]
Here's a screen recording that might help illustrating the bug: https://drive.google.com/file/d/1S-rB--3PYQlg6bZfsniNY9-NJyYN2jXK/view?usp=sharing

Notice that after removing the conversation, Open WebUI will immediately start a duplicated conversation.

**Reply by tjbck** [reply]
Fixed on dev!
